ipv-stub-template.yml

```diff
@@ -33,13 +33,30 @@ Globals:
       - !Ref AWS::NoValue
 
 Resources:
+  DynamoDB:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: FormResponse
+      KeySchema:
+        - AttributeName: TestId
```

Comment:
Would `FormId` be a more suitable name?

Reply:
This is a placeholder - as im not sure what the structure would be.  DynamoDb require this on setup

Reply:
Oh actually we're going to store the form response against auth code so maybe the hash key should be `AuthCode`

Reply:
Updated to `FormId`